model/task/db.go

```diff
@@ -302,9 +302,11 @@ func ByTimeRun(startTime, endTime time.Time) db.Q {
 // ByTimeStartedAndFailed returns all failed tasks that started between 2 given times
 func ByTimeStartedAndFailed(startTime, endTime time.Time, commandTypes []string) db.Q {
 	query := bson.M{
-		StartTimeKey: bson.M{"$lte": endTime},
-		StartTimeKey: bson.M{"$gte": startTime},
-		StatusKey:    evergreen.TaskFailed,
+		"$and": []bson.M{
```

Comment:
Correct me if this isn't what we intended the logic here to be, but it's a bug I found for the commit queue query.